Android.bp

```diff
@@ -30,6 +30,8 @@ cc_defaults {
         "-DDLT_LIB_USE_UNIX_SOCKET_IPC",
         "-DCONFIGURATION_FILES_DIR=\"/vendor/etc\"",
         "-DDLT_USER_IPC_PATH=\"/dev/socket\"",
+        "-DDLT_WRITEV_TIMEOUT_SEC=1",
+        "-DDLT_WRITEV_TIMEOUT_USEC=0",
```

Comment:
Hello @slawomiradamczyk ,
can you remove the configuration change from the Android.bp? I assume that this is unrelated to the the compile issue.

I would then merge this PR, as we are currently not able upstream our changes due to limited team capacity.

Reply:
you are right. i missed that one. Thanks for your contribution and patience.